R/import_tag.R

```diff
@@ -33,10 +40,18 @@ import_tag <- function(
 
     name <- paste0("status_url_", tag_id)
 
-    status_url <<- glue("https://api.reach.vote/api/v1/imports/tags/{datareturn$id}")
+    status_url <<- glue("https://api.reach.vote/api/v1/imports/tags/{datareturn$data$id}")
+    ### Comments
+    # changed {datareturn$id} to {datareturn$data$id} 
+    # {datareturn$id} returns empty item
+    
+    # using superassignment operator means that status_url appears in environment in addition to intended item status_url_{tag_id}
+    # is superassignment needed here? extra status_url item may be confusing for user
```

Comment:
you're totally right, we should remove that. I probably developed this over a few sessions and sloppily left that in.